pkg/login/social/generic_oauth_test.go

```diff
@@ -87,7 +87,7 @@ func TestSearchJSONForEmail(t *testing.T) {
 		for _, test := range tests {
 			provider.emailAttributePath = test.EmailAttributePath
 			t.Run(test.Name, func(t *testing.T) {
-				actualResult := provider.searchJSONForAttr(test.EmailAttributePath, test.UserInfoJSONResponse)
+				actualResult, _ := provider.searchJSONForAttr(test.EmailAttributePath, test.UserInfoJSONResponse)
```

Comment:
```suggestion
				actualResult, err := provider.searchJSONForAttr(test.EmailAttributePath, test.UserInfoJSONResponse)
				require.NoError(t, err)
```

Reply:
I tried this, but in some cases `EmailAttributePath` is empty, so error returned.

Reply:
OK, I will have a look.